src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.quarto.component.spec.ts

```diff
@@ -1997,4 +1997,5 @@ describe('OnlineGameWrapperComponent of Quarto:', () => {
     });
 
 });
+// It should display "JeanJaja (152)" so we know how bad JeanJaja is bad at P4 !
```

Comment:
sesse tu un todo?

Reply:
ouiiii le Math.float haha :D

Reply:
Math.floor ouais, TODO à faire donc !

Reply:
Fait dans Part.ts !

public getPlayerZeroFloorElo(): number {
        return Math.floor(this.data.playerZeroElo);
    }